src/debuglog.js

```diff
@@ -42,19 +42,19 @@ function debuglog (set) {
 	if (debugEnv === undefined) {
 		debugEnv = process.env.NODE_DEBUG || ''
 	}
-	set = set.toUpperCase()
-	if (!debugs[set]) {
-		if (debugEnv === '*' || new RegExp('\\b' + set + '\\b', 'i').test(debugEnv)) {
+	const upperSet = set.toUpperCase()
```

Comment:
Ahhh I see. Ok, well, let's keep it then. The linting rule definitely makes sense.